doc/source/howtos/upgrade_guide.rst

```diff
@@ -75,39 +75,44 @@ To upgrade the cluster, follow these steps for each node:
 
       sudo mysqld --skip-grant-tables --user=mysql --wsrep-provider='none'
 
+   .. note:: As of |PXC| 5.7.6, the ``--skip-grant-tables`` option
+      is not required.
+
 #. Open another session and run ``mysql_upgrade``.
 
-#. When the upgrade is done, stop ``mysqld``.
+#. When the upgrade is done, stop the ``mysqld`` process.
+   You can either run ``sudo kill`` on the ``mysqld`` process ID,
```

Comment:
Alexey Zhebel [1:09 PM] 
Hi Krunal


[1:09] 
running `kill` on a process without the `-9` option gracefully shuts the process down


[1:10] 
this is what Hrvoje does when testing the upgrade procedure (edited)


[1:11] 
It sends `SIGTERM` which should be pretty much what `mysqladmin shutdown` does